modules/commons/src/main/frontend/src/SearchBar.jsx

```diff
@@ -240,6 +241,15 @@ function SearchBar(props) {
                       <QuickSearchResult resultData={result} />
                     </MenuItem>
                   ))}
+                  { moreResults > 0 &&
+                    <MenuItem
+                      className={classes.dropdownItem}
+                      key="more"
```

Comment:
Maybe this will fix the popper-not-closing issue:

```suggestion
                      key="more"
                      onClick={() => setPopperOpen(false)}
```